be/src/storage/tablet_manager.h

```diff
@@ -47,6 +48,12 @@ namespace starrocks {
 class Tablet;
 class DataDir;
 
+enum TabletDropFlag {
+    kMoveFilesToTrash = 0,
```

Comment:
What's the difference between the three flags?

Reply:
Just as the name implies, `kMoveFilesToTrash` will move data files to trash, `kDeleteFiles` will delete files from disk directly, and `kKeptMetaAndFile` will leave the tablet meta and data files untouched.